src/screens/NetworkSettings/NetworkPreSettingsScreen.js

```diff
@@ -0,0 +1,104 @@
+import React from 'react';
+import {
+  StyleSheet,
+  View,
+  Text,
+  TouchableHighlight,
+} from 'react-native';
+import { t } from 'ttag';
+import HathorHeader from '@components/HathorHeader';
+import NewHathorButton from '@components/NewHathorButton';
+import { useDispatch } from 'react-redux';
+import { networkSettingsUpdateSuccess } from '../../actions';
+import { PRE_SETTINGS_MAINNET, PRE_SETTINGS_TESTNET } from '../../constants';
+
+const riskDisclaimerTitleText = t`Network Pre-Settings`.toUpperCase();
```

Comment:
Why did you use this variable name?

Reply:
- correct it to the right name: d1a61a7da2f21d01c9000f442e0199a593d1a048